getLinkHeaderURI.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"fmt"
+	"regexp"
+)
+
+/*
+ * Returns the URI in a link header with the specified "rel" attribute
+ * or an empty string when the "rel" attribute is not found. For example:
+ * link = `<https://example.com>; rel="test"`
+ * rel = `test`
+ * returns `https://example.com`
+ */
+func getLinkHeaderURI(link string, rel string) string {
```

Comment:
Whenever you make a paginated request to the API, it returns a link header that looks as follows:
```
link: <https://api.github.com/user/394452/events?per_page=100&page=1>; rel="prev", <https://api.github.com/user/394452/events?per_page=100&page=3>; rel="next", <https://api.github.com/user/394452/events?per_page=100&page=3>; rel="last", <https://api.github.com/user/394452/events?per_page=100&page=1>; rel="first"
```

Baiscally this is one big string, so getting the URL that belongs to `rel="next"` requires something like a parser or regex, which I wrote for this. It's very convenient to use this header, since you'll know you've reached the last page when `rel="next"` is not present anymore (`getLinkHeaderURI()` returns an empty string)